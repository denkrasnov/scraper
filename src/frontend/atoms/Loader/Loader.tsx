import React, { FC } from "react";

import styles from "./Loader.css";

const Loader: FC = () => {
  return (
    <div className={styles["lds-roller"]}>
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  );
};

export default Loader;
