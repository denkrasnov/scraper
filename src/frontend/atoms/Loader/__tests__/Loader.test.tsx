import React from "react";
import { shallow } from "enzyme";

import Loader from "..";

describe("Loader", () => {
  it("renders", () => {
    const component = shallow(<Loader />);

    expect(component).toMatchSnapshot();
  });
});
