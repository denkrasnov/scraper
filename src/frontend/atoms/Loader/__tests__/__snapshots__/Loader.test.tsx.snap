// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders 1`] = `
<div
  className="lds-roller"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
